components/frontend/src/metric/Target.js

```diff
@@ -48,22 +48,20 @@ function debtTargetActive(metric, direction) {
 }
 
 function ColoredSegment({ children, color, show, status }) {
-    const darkMode = useContext(DarkMode)
     if (show === false) {
         return null
     }
     return (
-        <Segment inverted color={color}>
-            <Segment inverted={darkMode}>
-                <Header>
-                    <span>
-                        {STATUS_SHORT_NAME[status]} <StatusIcon status={status} size="tiny" />
-                    </span>
-                    <Header.Subheader>{capitalize(color)}</Header.Subheader>
-                </Header>
-                <b>{children}</b>
-            </Segment>
-        </Segment>
+        <Box sx={{ padding: "10px", border: `12px solid ${STATUS_COLORS_MUI[status]}` }}>
```

Comment:
Why inline styling here?

Reply:
It's the recommended way to override styling on a one-off basis, see https://mui.com/material-ui/customization/how-to-customize.